server/src/test/java/io/crate/metadata/upgrade/MappingConstants.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Licensed to Crate.io GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.metadata.upgrade;
+
+public class MappingConstants {
+
+    /*
+    5.3 mapping for the table created with statement
+
+    create table novels (
+        title string,
+        description string,
+        author object as (
+            name string, birthday timestamp with time zone
+        ),
+        index title_desc_fulltext using fulltext(title, description),
+        index nested_col_fulltext using fulltext(title, author[ 'name' ]) with(analyzer = 'stop')
+    )
+
+    Has a column with multiple copy_to entries (column is part of many indices, in the new mapping has to be referenced in different sources)
+    Has a sub-column with copy_to, sources should be able to reference deep columns (must be FQN).
+    */
+    static final String FULLTEXT_MAPPING_5_3 = """
```

Comment:
Can we inline these to the test case where its used? There's only a single test case accessing it.

We tried sharing stuff like this before (`TableDefinitions`) but imho it's not worth it as each time you add a new test case you either need to figure out a way to make it work with the existing schema, or find a way to extend the schema without breaking test cases that use the schema already.

Reply:
it's used 2 tests (IndexTemplateUpgraderTest and MetadataIndexUpgraderTest)